### ISSUE TITLE
Clicking on JavaScript comment lines in HTML causes error

### ISSUE THREAD

**Issue by KyleMayes** [Microsoft/vscode#30774]
- VSCode Version: Code 1.14.1 (2648980a697a4c8fb5777dcfb2ab110cec8a2f58, 2017-07-13T19:18:47.188Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: none
---

Steps to Reproduce:

1. Create a file with the `HTML` language mode with the following contents:

```html
<html>
  <head>
    <script>
      // Comment.
    </script>
  </head>
  <body>
  </body>
</html>

```
2. Click on the line with the comment in the `<script>` tag.

3. The `Output` window will open with a message like the following:

```
[Error - 2:44:26 PM] Request textDocument/documentHighlight failed.
  Message: Request textDocument/documentHighlight failed with message: Cannot read property 'kind' of undefined
  Code: -32603 
```

**Comment by aeschli** [comment]
The failure is inside typescript which gets a content that only contains has a single line comment.
![image](https://user-images.githubusercontent.com/6461412/28258069-c360b706-6b01-11e7-885d-acb854cbc958.png)


text is:
```

            
      // Comment.
    


```

**Comment by mjbvz** [comment]
Opened https://github.com/Microsoft/TypeScript/issues/17245 to track this upstream

**Comment by xjy666** [comment]
How do you solve this problem?

**Comment by aeschli** [comment]
@xjy666 We are waiting for a fix in the typescript library... There is nothing you can do on your side.

**Comment by irkreja** [comment]
Please solve this ASAP.

**Comment by mjbvz** [comment]
Should be fixed in latest insiders which picked up TS 2.5
